Fix role so that it correctly handles Debian 11 (Bullseye)

jsf9k opened the pull request:
## 🗣 Description

This pull request modifies the role to correctly handle Debian 11 (Bullseye).  The `python-*` packages were recently removed from Debian 11 in favor of the more specific `python3-*`, and this broke the existing code.

## 💭 Motivation and Context

The Ansible role [cisagov/ansible-role-openvpn](https://github.com/cisagov/ansible-role-openvpn) has this role as a dependency.  When making modifications to that role I realized that it was failing because this role was failing due to the Debian 11 changes described in the previous section.

## 🧪 Testing

All pre-commit hooks and molecule tests pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

jsf9k committed:
Fix role for Debian Bullseye (11)

The python-* packages were recently removed from Debian 11 in favor of
the more specific python3-*.

jsf9k committed:
Update tests for changes related to Debian 11

jsf9k committed:
Add comments explaining why a release value of None is being handled

dav3r commented on molecule/default/tests/test_default.py:
```diff
@@ -31,9 +31,30 @@ def test_python_amazon(host, pkg):
     """Test that the appropriate packages were installed."""
+    # host.system_info.release can be None if the release is still a
+    # testing release that has not been officially released yet.  This
+    # is currently (2020/03/27) the case for Debian 11 (Bullseye).
```
👍👍 for this comment.

mcdonnnj replied:
Informative 👍

dav3r reviewed (approved):
I hit the bullseye every night! 
It's so easy, easy... 🎯

jsf9k committed:
Make paths variable more precise

In some cases, just leaving this as "vars" could cause this role to
pick up the vars files from a parent role that is using it.  I ran
into this specific error case today with cisagov/ansible-role-openvpn,
which has this role as a dependency.

mcdonnnj reviewed (approved):
Nice, this PR really hits the mark. 🎯
